### ISSUE TITLE
IS THIS A TROJAN?

### ISSUE THREAD

**Issue by Liamvazqz** [moom825/Discord-RAT-2.0#71]
All my AV say it's a trojan, can anyone help?

**Comment by A7QT** [comment]
yes, its a R.A.T itself luckily malwarebytes shut it down before it took effect on my pc.

**Comment by QuantumQuasar404** [comment]
Anything is possible, it is offering you good software but it may come at a cost. You don't know if something is running in the background when you open the executable, anything could be going on behind your computer just as they offer you to create a RAT and control it yourself, they already did it with your computer. It's been two years and it could very well be updating automatically and have an open session. Better not to risk it, and if you download it, do it in a virtual machine. ;)

**Comment by moom825** [comment]
you guys are special, there isnt that much code, just read it. and for the
compiled one, just use dnspy to read it. and just always use a vm for
something you dont trust.
